docs/swagger.yaml

```diff
@@ -1520,6 +1520,42 @@ paths:
       security:
         - access_token: []
   /tags:
+    get:
+      tags:
+        - tag
+      operationId: getTags
+      summary: Get all tags
+      description: Get all tags
+      responses:
+        '200':
+          description: A successful operation
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
@beblicarl since we're getting all tags, is it supposed to be an object or array of objects?

Reply:
you are right, i have implemented and pushed the changes